src/Symfony/Component/Validator/Constraints/Unique.php

```diff
@@ -30,6 +30,11 @@ class Unique extends Constraint
 
     public $message = 'This collection should contain only unique elements.';
 
+    /**
+     * @var string|callable
+     */
+    public $valueNormalizer;
```

Comment:
I think, on other constraints, this setting was simply called `$normalizer`.

Please also add a constructor argument.

Reply:
Done. Thanks!